src/main/java/uk/gov/hmcts/darts/annotation/controller/AnnotationController.java

```diff
@@ -31,4 +36,30 @@ public ResponseEntity<PostAnnotationResponse> annotationsPost(MultipartFile file
         var annotationId = annotationService.process(file, annotation);
         return ResponseEntity.ok(new PostAnnotationResponse(annotationId));
     }
+
+    @Authorisation(
+            bodyAuthorisation = true,
+            contextId = ANY_ENTITY_ID,
```

Comment:
we should use the annotation contextId here, but its not been merged yet. Tony is introducing it as part of https://github.com/hmcts/darts-api/pull/1032

Reply:
merged Tony's ticket, and used Annotation context_id